crates/cli/Cargo.toml

```
@@ -6,13 +6,18 @@ description = "CLI for Kora gasless relayer"
 license = "MIT"
 
 [[bin]]
-name = "kora-cli"
+name = "kora"
 path = "src/main.rs"
 
+[features]
+docs = ["kora-lib/docs", "dep:utoipa"]
+
 [dependencies]
 kora-lib = { path = "../lib", version = "1.0.2" }
 clap = { workspace = true }
 tokio = { workspace = true }
 env_logger = { workspace = true }
 solana-client = { workspace = true }
 dotenv = { workspace = true }
+serde_json = { workspace = true }
+utoipa = { workspace = true, optional = true }
```

Need to add tracing to set up logger:

```suggestion
utoipa = { workspace = true, optional = true }
tracing = { workspace = true }
tracing-subscriber = { workspace = true }
```